redisson/src/main/java/org/redisson/RedissonScoredSortedSet.java

```diff
@@ -204,6 +204,11 @@ public Integer addAndGetRevRank(double score, V object) {
         return get(addAndGetRevRankAsync(score, object));
     }
 
+    @Override
+    public Collection<Long> addAndGetAllRevRank(Map<? extends V, Double> map) {
```

Comment:
@mrniko not sure about this method signature, to stay in line with other methods it should be integer, however eval_list seems to be returning list of long instead of list of integer..

Reply:
see https://github.com/redisson/redisson/pull/2982#discussion_r471854691